### PR TITLE
noti: add page

### PR DESCRIPTION
**Pull request by AnimiVulpis**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/noti.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# noti
+
+> Monitor a process and trigger a notification.
```
Comment (sbrl) [inline pages/common/noti.md]: - Is it a graphical notification, or a textual one?
 - Does it show only when the command finishes executing?

File: pages/common/noti.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Display a notification when tar finishes compressing files:
+
+`noti {{tar -cjf example.tar.bz2 example/}}`
+
+- Display a notification even when you put it after the command to watch:
+
+`{{command_to_watch}}; noti`
+
+- Monitor a process by PID and trigger a notification when the pid disappears:
```
Comment (agnivade) [inline pages/common/noti.md]: The latter PID should be in caps.

File: pages/common/noti.md (hunk 1 above)
Comment (AnimiVulpis) [reply pages/common/noti.md]: Yes, it's graphical, using the default OS notification banner/popup

As far as I know it's supposed indicate a finished process, but I am pretty sure you could somehow plug it in the middle of a chain of commands (with some shell magic like `tee` perhaps?).

Should that, or parts of that, be mentioned in the description?

Comment (agnivade) [reply pages/common/noti.md]: Maybe not, since this is a tldr page. The basic usage should be fine.

Comment (AnimiVulpis) [reply pages/common/noti.md]: Then we should leave it like that

**Pull request merged**